raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -1055,14 +1055,14 @@ private boolean handleFetchResponse(
                         partitionResponse.snapshotId().endOffset(),
                         partitionResponse.snapshotId().epoch()
                     );
-                    return false;
+                    return true;
```

Comment:
Returning `false` seemed more appropriate. That ensures we will get a backoff before the next retry.

Reply:
Done.